txscript/data/script_tests.json

```diff
@@ -3,7 +3,7 @@
 ["It is evaluated as if there was a crediting coinbase transaction with two 0"],
 ["pushes as scriptSig, and one output of 0 atoms and given scriptPubKey,"],
 ["followed by a spending transaction which spends this output as only input (and"],
-["correct prevout hash), using the given scriptSig. All nLockTimes are 0, all"],
+["correct prevout hash), using the given scriptSig. All lock times are 0 and all sequences are max."],
```

Comment:
`sequences are max` should be on the next line replacing the `nSequences are max`.